LuaRules/Configs/airpad_defs.lua

```diff
@@ -0,0 +1,20 @@
+local airpadDefs = {
+	[UnitDefNames["factoryplane"].id] = {
+		padPieceName = {"land"}
+	},
+	[UnitDefNames["staticrearm"].id] = {
+		padPieceName = {"land1","land2","land3","land4"}
+	},
+	[UnitDefNames["shipcarrier"].id] = {
+		padPieceName = {"LandingFore","LandingAft"}
+	},
+}
+
+for unitDefID, config in pairs(airpadDefs) do
+    local ud = UnitDefs[unitDefID]
+
+    config.mobile = (not ud.isImmobile)
+    config.cap = config.cap or ud.customParams.pad_count
```

Comment:
I think I'd do just
```suggestion
    config.cap = ud.customParams.pad_count
```
because having two sources can be confusing.

Reply:
I will make it default to #padPieceName instead (because they are closely related in the code)

Reply:
Done.